src/core/platforms/android/androidplatformutilities.cpp

```diff
@@ -754,13 +754,13 @@ void AndroidPlatformUtilities::vibrate( int milliseconds ) const
 
 void AndroidPlatformUtilities::requestBackgroundPositioningPermissions()
 {
-  checkAndAcquirePermissions( QStringLiteral( "android.permission.ACCESS_BACKGROUND_LOCATION" ) );
+  checkAndAcquirePermissions( QStringList() << QStringLiteral( "android.permission.ACCESS_BACKGROUND_LOCATION" ) );
```

Comment:
```suggestion
  checkAndAcquirePermissions( { QStringLiteral( "android.permission.ACCESS_BACKGROUND_LOCATION" ) } );
```

Do initializer lists work here?

Reply:
@m-kuhn , let me check.

Reply:
@m-kuhn , it does work.